etna/lib/etna/client.rb

```diff
@@ -147,7 +147,10 @@ def request(uri, data)
             yield response
           end
         else
-          Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
+          verify_mode = @ignore_ssl ?
+            OpenSSL::SSL::VERIFY_NONE :
+            OpenSSL::SSL::VERIFY_PEER
+          Net::HTTP.start(uri.host, uri.port, use_ssl: true, verify_mode: verify_mode) do |http|
```

Comment:
We also need to pass through the `ignore_ssl` flag for non-persistent connections, otherwise development and staging environments don't work.

Reply:
Nice catch!